package.json

```diff
@@ -173,6 +173,7 @@
     "mmjstool": "github:mattermost/mattermost-utilities#3b4506b0f6b14fbb402f9f8ef932370e459e3773",
     "mocha-circleci-reporter": "0.0.3",
     "mochawesome": "7.1.3",
+    "node-abi": "3.25.0",
```

Comment:
(question) Is the explicit dependency needed here? All the packages that need it have it in their dependencies (e.g. electron-rebuild etc). Is this needed to pin it to a specific version?

Reply:
We should be able to remove this, but for now it's needed because of how new Electron v21 is. Some of the downstream dependencies are using older versions of `node-abi` so we need to make sure we have the latest.